tests/foreman/api/test_role.py

```diff
@@ -100,7 +100,48 @@ def test_positive_update(self):
 class CannedRoleTestCases(APITestCase):
     """Implements Canned Roles tests from API"""
 
-    @stubbed()
+    def create_org_admin(self, name=None, orgs=None, locs=None):
+        """Helper function to create org admin role for particular
+        organizations and locations by cloning 'Organization admin' role.
+
+        :param str name: The name of cloned Org Admin role
+        :param list orgs: The list of organizations for which the org admin is
+            being created
+        :param locs: The list of locations for which the org admin is
+            being created
+        :return dict: This function returns dict representation of cloned role
+            data returned from 'clone' function
+        """
+        name = gen_string('alpha') if not name else name
+        orgs = [] if not orgs else orgs
+        locs = [] if not locs else locs
```

Comment:
There's shorter form with no extra vars:
```
                'role': {
                    'name': name,
                    'organization_ids': orgs or [],
                    'location_ids': locs or [],
                }
```
If orgs or locs is `None` - `[]` will be selected instead

Reply:
Must only be careful when the var is a number because 0 will be evaluated to False and some times you just want to filter None. But in case of a list, this is fine.

Reply:
Done.